helpers/__init__.py

```diff
@@ -0,0 +1,5 @@
+from helpers.http_session import http
+
+__all__ = [
```

Comment:
do you have a link to this pattern? looks cool never seen before

Reply:
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package